### ISSUE TITLE
[Docs]Error Hyperlinks in Contribute.d

### ISSUE THREAD

**Issue by Ghat0tkach** [zowe/zowe.github.io#833]

**Comment by balhar-jakub** [comment]
What links doesn't work for you? 

I was testing all of the links on my computer and they works. Is it possible that there is a proxy setting, that prevents you from accessing them?

**Comment by Ghat0tkach** [comment]
> What links doesn't work for you? 
> 
> I was testing all of the links on my computer for https://www.zowe.org/contribute and they works. Is it possible that there is a proxy setting, that prevents you from accessing them?

Oh actually i was trying to access them from contribute.md document from the repo itself

**Comment by balhar-jakub** [comment]
That is correct as the actual links are stored as information in data and needs to be preprocessed first for the publication on the zowe.org which is the target for this repository. 

We may need to add some more information about this to the readme.md, though.

**Comment by Ghat0tkach** [comment]
Then if that is the case , may i know which sort of information shall be added on readme.md? I'll change this issue to that specific one

**Comment by balhar-jakub** [comment]
Maybe just add something in this section: https://github.com/zowe/zowe.github.io#welcome-to-zowe-website-development

Something like this:
The links and content in the .md files need to be preprocessed and published on zowe.org before they're usable. If you try clicking on the links directly from the repository, they won't work.

**Comment by Ghat0tkach** [comment]
Okay Thank You and 
any plans for issue and Pull request templates?
